.storybook/preview.js

```diff
@@ -1,5 +1,8 @@
 import { addDecorator } from '@storybook/react';
 import themeDecorator from './decorators';
+import { MemoryRouter } from "react-router";
+    
+addDecorator(story => <MemoryRouter initialEntries={['/']}>{story()}</MemoryRouter>);
```

Comment:
I added this when i was working on the organism carousel, if you ever need a story where you are playing around with routes / <Link /> component this resolves errors of not having <Route> wrapper around the app. I think its worth keeping it so that we dont have to dig into why stuff isnt working with routes